SimplenoteUITests/NoteList.swift

```diff
@@ -129,6 +129,28 @@ class NoteList {
         guard searchCancelButton.exists else { return }
         searchCancelButton.tap()
     }
+
+    class func longPressNote(title: String) {
+        app.tables.cells[title].press(forDuration: 3)
+    }
+
+    class func selectNote() {
+        let selectButton = app.buttons[UID.Button.select]
+        guard selectButton.exists else { return }
+        selectButton.tap()
+    }
+
+    class func selectAll() {
+        let selectAllButton = app.buttons[UID.Button.selectAll]
+        guard selectAllButton.exists else { return }
+        selectAllButton.tap()
+    }
+
+    class func tapTrashButton() {
```

Comment:
```suggestion
    class func tapTrashNotesButton() {
```

to better reflect that this is about trashing multiple notes, and not about trashing a single note by swiping it left ad then tapping the `Trash` button.